src/components/organisms/Tabs/TabButtons/__snapshots__/TabButtons.test.tsx.snap

```diff
@@ -145,7 +145,7 @@ exports[`<TabButtons /> renders the buttons on big screens 1`] = `
         <button
           aria-controls="pineapple-content"
           aria-expanded="true"
-          class="c2 c3 py-2 sc-cQFLBn c3 py-2"
+          class="c2 c3 py-2 sc-gojNiO c3 py-2"
```

Comment:
I'm not sure why these snapshots were out of date 🤔

Reply:
Interesting. If you run the tests again does it generates a new ID? do you have any idea where this class is coming from?

Reply:
Yeah, I re-installed my node modules and got a different ID... any ideas?

Reply:
Yeah, I re-installed my node modules and got a different ID... any ideas?

Reply:
Okay, I just deleted & cloned down the whole repo again and got the same thing. This is odd.

Reply:
Let's leave it for now and see if it happens to more people